includes/Core/Assets/Manifest.php

```diff
@@ -38,7 +38,7 @@ class Manifest {
 	 */
 	public static function get_filename( $handle ) {
 		if ( is_null( self::$assets ) ) {
-			$path = Plugin::instance()->context()->path( "dist/manifest.php" );
+			$path = Plugin::instance()->context()->path( 'dist/manifest.php' );
```

Comment:
Previously this class was being ignored by the phpcs config and I noticed it was still ignored (even though it's no longer generated) when phpcs was failing on the now generated `dist/manifest.php` file (which is now ignored 😄 )